tensorboard/webapp/widgets/range_input/range_input_test.ts

```diff
@@ -192,6 +193,16 @@ describe('range input test', () => {
 
         expect(getRangeThumbsStyleLeft(fixture)).toEqual(['50%', '50%']);
       });
+
+      it('does not render slider when min == max', () => {
```

Comment:
You should probably also add a test that it does show slider-track when min and max are different.

Reply:
There are a bunch of tests which interact with the slider. I think that should be sufficient.